src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/QueryStringValidation.feature

```diff
@@ -0,0 +1,186 @@
+            """
+            """
+    Then there is a response header with "totalCount" equals to 0
+
+  @ignore
+  Scenario: Ensure that schools return the total count
+    Given these schools exist:
+            | schoolId     | nameOfInstitution                    | gradeLevels                                                            | educationOrganizationCategories      |
+            | 5            | School with max edorgId value        | [ "Postsecondary" ]                                                    | [ "School" ]                         |
+            | 255901001    | Grand Bend High School               | [ "Tenth grade", "Ninth grade", "Eleventh grade", "Twelfth grade" ]    | [ "School" ]                         |
+    When sending a GET request to "/ed-fi/schools?totalCount=true" 
+            """
+            """
+    Then there is a response header with "totalCount: 2" 
+
+  @ignore
+  Scenario: Ensure that schools return the total count
+    Given these schools exist:
+            | schoolId     | nameOfInstitution                    | gradeLevels                                                            | educationOrganizationCategories      |
+            | 5            | School with max edorgId value        | [ "Postsecondary" ]                                                    | [ "School" ]                         |
+            | 255901001    | Grand Bend High School               | [ "Tenth grade", "Ninth grade", "Eleventh grade", "Twelfth grade" ]    | [ "School" ]                         |
+    When sending a GET request to "/ed-fi/schools?totalCount=false" 
+            """
+            """
+    Then it should respond with 200
+    And header "totalCount" is not present
+
+  @ignore
+  Scenario: Ensure that schools return the total count
+    Given these schools exist:
+            | schoolId     | nameOfInstitution                    | gradeLevels                                                            | educationOrganizationCategories      |
+            | 5            | School with max edorgId value        | [ "Postsecondary" ]                                                    | [ "School" ]                         |
+            | 255901001    | Grand Bend High School               | [ "Tenth grade", "Ninth grade", "Eleventh grade", "Twelfth grade" ]    | [ "School" ]                         |
+    When sending a GET request to "/ed-fi/schools" 
+            """
+            """
+    Then it should respond with 200
+    And header "totalCount" is not present
+
+
+  @ignore
+  Scenario: Ensure clients can GET information changing the casing of the query value to be all lowercase
+    Given this school exists:
+            | schoolId     | nameOfInstitution                    |                                                          | educationOrganizationCategories      |
+            | 5            | School with max edorgId value        |
```

Comment:
This line is missing the third column: `educationOrganizationsCategories`

The entire table is missing the `gradeLevels` column.

Reply:
Done